01_part-to-whole_cnell/_targets.R

```diff
@@ -0,0 +1,51 @@
+library(targets)
+library(tidyverse)
+library(rvest)
+library(spData)
+library(sf)
+library(cartogram)
+library(showtext)
+library(scico)
+library(gganimate)
+
+options(tidyverse.quiet = TRUE)
```

Comment:
This command is actually not doing what is trying to at the moment. For organizing targets pipeline packages, you would usually have `library()` calls here for any package that is used to setup/define a target. Any package used while _building_ the targets should go after this line in a `tar_option_set()` call. Then the tidyverse message will actually be hidden and the packages will only be loaded while used within the build environment.

```suggestion
options(tidyverse.quiet = TRUE)
tar_option_set(packages = c('tidyverse', 'rvest', 'spData', 'sf', 'cartogram',
                            'showtext', 'scico', 'gganimate'))
```

Reply:
When trying to build the pipeline, I got an error during the final target: `Error: callr subprocess failed: transformr is required to tween sf layers`. So I think you should add `transformr` to this package list.